modules/start_on_team_color/functions/SOTC/fn_SOTC_Init.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+if (!isDedicated) then {
+  #include "..\..\settings.sqf"
+
+  "" spawn {
+    private _color = GETVAR(player, FW_TeamColor, "NONE");
```

Comment:
The SETVAR and GETVAR macros check for the prefix already, so second input should just be `teamColor`

Reply:
Done.